### PR TITLE
chore: simplify namings

### PR DESCRIPTION
**Pull request by Lightning00Blade**

### CODE REVIEW

File: src/bidiMapper/domains/script/PreloadScriptStorage.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class PreloadScriptStorage {
   readonly #scripts = new Set<PreloadScript>();
 
   /** Finds all entries that match the given filter. */
-  findPreloadScripts(filter?: BidiPreloadScriptFilter): PreloadScript[] {
+  find(filter?: BidiPreloadScriptFilter): PreloadScript[] {
```
Comment (Lightning00Blade) [inline src/bidiMapper/domains/script/PreloadScriptStorage.ts:34]: Used as PreloadScriptStorage.findPreloadScript double points to PreloadScript.

Comment (thiagowfx) [reply src/bidiMapper/domains/script/PreloadScriptStorage.ts:34]: LGTM, but:

`find` <-> `remove`

`findAll` <-> `removeAll`

It seems odd to have one of them with `All`, but not the other.

I feel `removeAll` -> `remove` would be a better change.

Comment (Lightning00Blade) [reply src/bidiMapper/domains/script/PreloadScriptStorage.ts:34]: Nice catch, I missed the `s` at the end and type. Agree `remove` would be better

Comment (thiagowfx) [inline src/bidiMapper/domains/script/PreloadScriptStorage.ts:34]: `BidiPreloadScriptFilter` -> `PreloadScriptFilter`

**Review by thiagowfx — COMMENTED** [review]
LGTM once requested changes are applied

File: src/bidiServer/reader.ts [hunk 2]
```diff
@@ -18,6 +18,6 @@
 import fs from 'fs/promises';
 import path from 'path';
 
-export default async function read(): Promise<string> {
+export async function readMapperTabFile(): Promise<string> {
```
[the remark this answers is not in the document]
Comment (thiagowfx) [reply src/bidiServer/reader.ts:21]: No concerns here.

**Review by thiagowfx — APPROVED** [review]

**Pull request merged**